user_templates/mcu_defaults/main/Fysetc_Spider_v3.x.cfg

```diff
@@ -39,14 +39,14 @@ aliases:
     BED_HEATER=MCU_BED_OUT , BED_TEMPERATURE=MCU_TB ,
 
     PART_FAN=MCU_FAN1 , E_FAN=MCU_FAN0 ,
-    CONTROLLER_FAN=MCU_FAN2        ,
-    EXHAUST_FAN=MCU_HEAT2          ,
-    FILTER_FAN=MCU_RGB_R           ,
-    HOST_CONTROLLER_FAN=MCU_RGB_G  ,
+    CONTROLLER_FAN=MCU_FAN2       ,
+    EXHAUST_FAN=MCU_HEAT2         ,
+    FILTER_FAN=MCU_FAN4           ,
+    HOST_CONTROLLER_FAN=MCU_FAN3  ,
 
     CHAMBER_TEMPERATURE=MCU_TE1 , ELECTRICAL_CABINET_TEMPERATURE=MCU_TE2 ,
 
     LIGHT_OUTPUT=MCU_HEAT1        ,
-    LIGHT_NEOPIXEL=MCU_RGB_B      ,
+    LIGHT_NEOPIXEL=MCU_FAN5       ,
```

Comment:
I'm pretty sure it's not possible to configure a neopixel band on a FAN port. Can you revert this one to the RGB port?

For the others it's ok.